docs/help/custom-domains.md

```diff
@@ -1,23 +1,28 @@
 # Custom domains
 
-## Before you begin
+Your docs are hosted on Bump.sh. However you can set a custom domain: visitors of your docs will stay within your brand experience. For this, you'll use a CNAME.
```

Comment:
> your docs will stay within your brand experience

Brand sounds a bit too commercial to me, visitors can also be from the organization.

Reply:
I completely agree with Jay remark. Why do we need to talk about brand here?

Reply:
```suggestion
Your docs are hosted on Bump.sh. However you can set a custom domain: visitors of your docs will then stay within your own name. For this, you'll use a CNAME.
```

Reply:
I've moved the suggestion to #66